static/app/utils/analytics/issueAnalyticsEvents.tsx

```diff
@@ -39,6 +39,11 @@ export type IssueEventParameters = {
     search_source: string;
     search_type: string;
   };
+  'issues_stream.count_perf_issues': {
+    num_perf_issues: number;
```

Comment:
you should add the total as well in case we want to check % of perf issues on a page.

Reply:
done